code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -122,6 +121,8 @@
 		CAN_MOVE_DIAGONALLY = FALSE
 		user.eyeobj.glide_size = DELAY2GLIDESIZE(1)
 		user.delayNextMove(1)
+	else if(user.currently_tracking)
+		user.eyeobj.glide_size = user.currently_tracking.glide_size
```

Comment:
Glide size doesn't seem to do anything unless the AI actively has the client call Move, so this doesn't really work when tracking from what I can tell

Reply:
i have noticed this in testing